src/engraving/libmscore/stemslash.cpp

```diff
@@ -57,7 +58,7 @@ void StemSlash::draw(mu::draw::Painter* painter) const
 void StemSlash::setLine(const LineF& l)
 {
     line = l;
-    double w = score()->styleMM(Sid::stemWidth) * .5;
+    double w = _width * .5;
     setbbox(RectF(line.p1(), line.p2()).normalized().adjusted(-w, -w, 2.0 * w, 2.0 * w));
 }
```

Comment:
Since this method is called in exactly one place, what do you think about just inlining this code in the layout() method, and getting rid of the _width member? (Or do you foresee the method might be useful in other places in the future?)

Reply:
Yeah I actually thought the same, gonna do that